addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -61,6 +61,15 @@ const LinkPopoverWidget = Widget.extend({
         .popover('show')
         .data('bs.popover').tip.classList.add('o_edit_menu_popover');
 
+        // hide popover while typing on mega menu
+        if (this.$target.parents('.o_mega_menu').length) {
+            const showPopoverDebounced = _.debounce(() => this.$target.popover('show'), 1500);
```

Comment:
Mmh, using debounce for this seems a bit overkill, use setTimeout after keydown and clear it at each keydown. Also, the less dependence to underscore the better.

Reply:
Meh. Ok, I though it was cleaner that way :)